docs/joss_paper/paper.md

```diff
@@ -0,0 +1,150 @@
+---
+title: 'Snek5000: Python framework for Nek5000'
+tags:
+  - CFD
+  - Python
+  - Nek5000
+  - FluidSim
+  - Snakemake
+authors:
+
+    name: Ashwin Vishnu Mohanan
+    orcid: 0000-0002-2979-6327
+    equal-contrib: true
+    affiliation: 1
+
+    name: Arman Khoubani
+    orcid: 0000-0002-0295-5308
+    equal-contrib: true
+    affiliation: 2
+
+    name: Pierre Augier
+    orcid: 0000-0001-9481-4459
+    equal-contrib: true
+    affiliation: 2
+
+affiliations:
+
+    name: Swedish Meteorological and Hydrological Institute, Norrk{\"o}ping, Sweden
+    index: 1
+
+    name: Laboratoire des \'Ecoulements G\'eophysiques et Industriels, Universit\'e~Grenoble~Alpes, 
+      CNRS, Grenoble~INP, 38000~Grenoble, France
+    index: 2
+
+date: 2 March 2023
+bibliography: paper.bib
+---
+
+# Summary
+
+TODO: add a lot of citations, in particular of software papers...
+
+Computational fluid dynamics (CFD) simulations are essential tools in various scientific
+and engineering disciplines. Nek5000 is a CFD Fortran code based on spectral element
+methods. It has a proven track record in numerous applications. In this article, we
+present Snek5000, a Python package designed to streamline the management and
+visualization of fluid dynamics simulations based on Nek5000. The package builds upon
+the functionality of Nek5000 by providing a user-friendly interface for launching and
+restarting simulations, loading simulation data, and generating figures and movies. This
+paper introduces Snek5000, discusses its design principles, and highlights its impact on
+the scientific community.
+
+## Snek5000: features and capabilities
```

Comment:
It seems that summary has to be very short. So `Snek5000: features and capabilities` should not be in summary. I guess we should have a dedicated session.

```
# Snek5000: features and capabilities
```